cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -156,8 +156,8 @@ <h3 class="o-post-preview_title">
                     {{ tags.render(post.tags.names(), url, true, false, form_id) }}
                 {% endif %}
             </div>
-            {% if post.preview_link_text %}
-                <a href="{{ get_page_state_url({'request':request}, page) }}" class="jump-link jump-link__underline">
+            {% if post.preview_link_url and post.preview_link_text %}
+                <a href="{{ post.preview_link_url }}" class="jump-link jump-link__underline">
```

Comment:
Content will always manually enter it because it'll usually be an additional link. The header will be hyperlinked to the post. (Fixed in #1467)